packages/zcli-apps/src/commands/apps/create.ts

```diff
@@ -42,6 +42,7 @@ export default class Create extends Command {
       CliUx.ux.action.stop('Uploaded')
       CliUx.ux.action.start('Deploying app')
       const { job_id } = await deployApp('POST', 'api/apps.json', upload_id, manifest.name)
+      console.error('job_id', job_id)
```

Comment:
Can I ask the reason of logging job_id as an error and why we need this log?

Reply:
@EmilyBie Good catch, i've removed it 👍